src/Components/IndexScene/IndexScene.tsx

```diff
@@ -180,6 +181,18 @@ export class IndexScene extends SceneObjectBase<IndexSceneState> {
     this._subs.add(timeRange.subscribeToState(this.limitMaxInterval(timeRange)));
   }
 
+  private setVariableOperators() {
+    const fieldsVar = getFieldsVariable(this);
+    // No wip set when this is called, so once the operations must be consistent for all filters
+    // @todo we'll need to split out the duration, bytes, and float filters out of the VAR_FIELDS variable and into a new variable if we want to keep comparison operations out of string, int, etc filters
```

Comment:
remove todo, create issue

Reply:
Friendly reminder to remove comment and add the issue.